ui/packages/services/src/config/config.type.ts

```diff
@@ -29,9 +29,17 @@ export interface UiConfig {
     contextPath: string;
 }
 
+export interface AuthConfig {
+    url: string;
+    realm: string;
+    clientId?: string;
```

Comment:
clientId is optional?

Reply:
Shouldn't be the case, I'll change this.